docs/sources/csv.md

```diff
@@ -97,29 +97,29 @@ Germany,700
 
 ## Columns
 
-Though your csv file have columns, specify them as columns manually and only that fields will be considered for display. Columns will appear in the same order you specify. Each column will have following properties
+Even if your CSV file has columns, you should specify them manually, and only those fields will be considered for display. The columns will appear in the order you specify. Each column will have the following properties:
 
-| Column   | Description                                     |
-| -------- | ----------------------------------------------- |
-| Title    | Title of the column when using the table format |
-| Selector | Column name in CSV file. Case sensitive         |
-| Format   | Format of the column                            |
+| Column   | Description                                      |
+| -------- | -----------------------------------------------  |
+| Title    | Title of the column when using the table format. |
+| Selector | Column name in CSV file. Case sensitive.         |
+| Format   | Format of the column.                            |
 
-If you don't specify any columns, then the infinity plugin will try to auto generate the columns and all the fields will be returned as string. (This auto generate columns feature only works with table format)
+If you don't specify any columns, then the Infinity plugin will try to auto generate the columns and all the fields will be returned as string. Please note that this auto generate columns feature only works with table format.
 
 ## CSV URL
 
-In the below example, we are going to convert the CSV URL `https://gist.githubusercontent.com/yesoreyeram/64a46b02f0bf87cb527d6270dd84ea47/raw/32ae9b1a4a0183dceb3596226b818c8f428193af/sample-with-quotes.csv` into a grafana datasource.
+In the following example, we are going to convert the CSV URL `https://gist.githubusercontent.com/yesoreyeram/64a46b02f0bf87cb527d6270dd84ea47/raw/32ae9b1a4a0183dceb3596226b818c8f428193af/sample-with-quotes.csv` into a Grafana data source.
 
-CSV data should have columns as its first line and comma delimited. You need to also specify the column names manually for display purposes.
+CSV data should have columns as its first line and be comma delimited. You also need to specify the column names manually for display purposes.
 
 ![image](https://user-images.githubusercontent.com/153843/92382040-d8a35480-f103-11ea-8ff8-48c7ca211062.png#center)
 
-Ignore the root / rows as each line of CSV will be your rows.
+You can ignore the root/rows as each line of CSV will be your rows.
```

Comment:
Not sure how to re-word this one actually but it shouldn't use the slash.

Probably can leave for another time.

Reply:
we can probably fix this once we do a PR with Vale suggestions